### PR TITLE
Add feature of navigation between folders

### PR DESCRIPTION
**Pull request by kuzmavictor**
During this iteration was implemented:
- the functionality  to navigate between folders ;
- the field to display the current folder;
- the button to redirect on root folder.

### CODE REVIEW

**Comment by kuzmavictor** [comment]
@dkalinin @AndriiSameliuk PTAL.

File: app/view/component/page/file-list/index.js [hunk 1]
```diff
@@ -236,6 +250,16 @@ export class FileListPage extends ConfigComponent {
...
     }
   }
+  /**
+   * Shows current folder to file list
+   *
+   * @param {FileModel} folder - a current folder from state.
+   * @private
+   */
+  _showCurrentFolder(folder) {
+    this._currentFolderName.text = folder.name;
```
Comment (dkalinin) [inline app/view/component/page/file-list/index.js]: Let's do this in setter of _currentFolder.

File: app/view/component/file-item/index.js [hunk 2]
```diff
@@ -78,13 +79,14 @@ export class FileItem extends DraggableComponent {
   _renderFileDisplay() {
     this._fileNameSpan = new Span(this._htmlDivElement);
     this._fileNameSpan.text = this._name;
-    this._fileNameSpan.onDoubleClick((event)=>this._onDoubleClick(event));
+    this._fileNameSpan.onDoubleClick((event) => this._onDoubleClick(event));
```
Comment (dkalinin) [inline app/view/component/file-item/index.js]: Is this used? We are handling double-click in _onClick method.

File: app/service/store/action/index.js [hunk 3]
```diff
@@ -189,4 +189,15 @@ export const actions = {
...
+
+  /**
+   * Sets current folder.
+   *
+   * @param {Store} store - an instance of {@link Store}
+   * @return {Function}
+   */
+  setCurrentFolder: (store) => (folder) => {
+    store.commit('setCurrentFolder', folder);
+    store.dispatch('fetchFiles', store.state.currentFolder);
```
[the remark this answers is not in the document]
Comment (kuzmavictor) [reply app/service/store/action/index.js]: Request service  returns `Promises`. Inside of this action service is not called.

File: app/service/store/action/index.js [hunk 4]
```diff
@@ -188,4 +188,15 @@ export const actions = {
...
```
Comment (dkalinin) [inline app/service/store/action/index.js]: Let's return this since actions return promises.
fetchFiles returns promise.

**Pull request merged**